package.json

```diff
@@ -162,6 +162,7 @@
     "shelljs": "^0.8.5",
     "shx": "^0.3.2",
     "sinon": "^11.1.1",
+    "sinon8": "npm:sinon@^8",
     "typescript": "^4.8.4",
     "uglify-js": "^3.6.0",
     "unified": "^7.0.2",
```

Comment:
Note that we need to update the `videojs-generate-karma-config` version before this will work.